py/dml/messages.py

```diff
@@ -1707,6 +1712,31 @@ def log(self):
                 "implementation candidate provided by ancestor template "
                 + f"'{ancestor.name}'")
 
+class EMEMBERTQMIC(DMLError):
+    """A template-qualified method implementation call can only be done if
+    the specified template actually does provide or inherit an implementation
+    of the named method for the object instantiating the template. That the
+    template provides or inherits an abstract declaration of the method is not
+    sufficient.
+
+    Besides more mundane causes (e.g. misspellings), this error could happen if
```

Comment:
I think `Beside`, or `Apart from`/`Except for`, (cf "Besides, ..." which would give a parse error)

Reply:
Going with "Apart from", though I'm 90% sure that "Besides" (and not "Beside") is correct in this context.